templates/web/src/query.ts.twig

```diff
@@ -0,0 +1,37 @@
+class Query {
+  static equal(attribute: string, value: any): string {
```

Comment:
I would replace `any` with the actual types we allow 🤔 

Lets add a type like this 
```ts
type QueryTypes = string | string[] | number | number[] | boolean | boolean[];
```

and we can use it like this
```ts
static equal(attribute: string, value: QueryTypes) {

}
```